WordPress/src/main/res/values/strings.xml

```diff
@@ -599,7 +595,65 @@
     <string name="older_month">Older than a month</string>
     <string name="error_notification_open">Could not open notification</string>
     <string name="ignore">Ignore</string>
-    <string name="push_auth_expired">The request has expired. Sign in to WordPress.com to try again.</string>
+    <string name="push_auth_expired">The request has expired. Sign in to WordPress.com to try again.</string><string name="notifications_updates_from_wordpress">Updates from WordPress.com</string>
+
+    <!-- Notifications Settings -->
+    <string name="notification_settings">Notification Settings</string>
+    <string name="notifications_sights_and_sounds">Sights and Sounds</string>
+    <string name="your_sites">Your Sites</string>
+    <string name="notifications_account_emails_summary">We’ll always send important emails regarding your account, security, privacy, and purchase transactions, but you can get some fun extras, too!</string>
+    <string name="notifications_other_blogs_summary">Notification settings for your comments on other sites.</string>
+    <string name="notifications_account_emails">Updates from WordPress.com</string>
+    <string name="notifications_wpcom_updates">WordPress.com Updates</string>
+    <string name="notifications_other_blogs">Other Sites</string>
+    <string name="notifications_comments_other_blogs">Comments on other sites</string>
+    <string name="notifications_tab">Notifications tab</string>
+    <string name="email">Email</string>
+    <string name="app_notifications">App notifications</string>
+    <string name="comment_likes">Comment likes</string>
+    <string name="replies_to_your_comments">Replies to your comments</string>
+    <string name="error_loading_notifications">Could not load notifications settings</string>
```

Comment:
"Notifications" should be singular to match the activity title ("Notification Settings").